tests/ProcessManager/SeleniumManagerTest.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file is part of the Panthère project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Panthere\Tests\ProcessManager;
+
+use Panthere\ProcessManager\ChromeManager;
+use Panthere\ProcessManager\SeleniumManager;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @author Kévin Dunglas <dunglas@gmail.com>
```

Comment:
Should be changed I guess!

Reply:
I can't :(
https://github.com/dunglas/panthere/blob/master/.php_cs.dist#L6

Reply:
It defines the copyright, not the author! You can update this line safely, PHP CS Fixer will not complait.